x/wasm/keeper/msg_dispatcher.go

```diff
@@ -81,6 +82,18 @@ func (d MessageDispatcher) dispatchMsgWithGasLimit(ctx sdk.Context, contractAddr
 func (d MessageDispatcher) DispatchSubmessages(ctx sdk.Context, contractAddr sdk.AccAddress, ibcPort string, msgs []wasmvmtypes.SubMsg) ([]byte, error) {
 	var rsp []byte
 	for _, msg := range msgs {
+
+		// Prevent the use of fswap and fbridge module in Submessages
+		// https://github.com/Finschia/finschia-sdk/pull/1336, https://github.com/Finschia/finschia-sdk/pull/1340
+		if stargateMsg := msg.Msg.Stargate; stargateMsg != nil {
+			if strings.HasPrefix(stargateMsg.TypeURL, "/lbm.fswap.v1") {
+				return nil, sdkerrors.Wrap(types.ErrUnsupportedForContract, "fswap not supported by Stargate")
+			}
+			if strings.HasPrefix(stargateMsg.TypeURL, "/lbm.fbridge.v1") {
+				return nil, sdkerrors.Wrap(types.ErrUnsupportedForContract, "fbridge not supported by Stargate")
+			}
+		}
```

Comment:
How about extracting the deny list and adding only checking codes in this place?

Reply:
Is this proposal suggesting that we should adopt this approach in consideration of the increasing number of TypeURLs to check in the future?

Reply:
I don't know the future possibility. But I suggested a way to handle it efficiently.